src/ax/validation/inboundstage.go

```diff
@@ -0,0 +1,40 @@
+package validation
+
+import (
+	"context"
+
+	"github.com/jmalloc/ax/src/ax/endpoint"
+)
+
+// InboundStage is the validation stage in the
+// message inbound pipeline
+type InboundStage struct {
+	Validators []endpoint.Validator
+	Next       endpoint.InboundPipeline
+}
```

Comment:
I've been struggling to think of a good name for these pipeline stages. I've generally tried to provide names that describe what they do, rather than just saying that they're a stage.

 Obviously you get some context from the package name, but I'd still like to try to come up with something else. `outbox.Deduplicator` is a pretty good example of this I think. Maybe these are "rejecters"... `validation.InboundRejecter` / `OutboundRejecter` ? That's not great either, but we'll come up with something.

Reply:
Can these structs be `validation.InboundVerifier` and  `validation.OutboundVerifier`?

Reply:
I don't like introducing the word 'verify' when it's 'validate' elsewhere, but if you mean `InboundValidator`, etc that's probably better than `InboundStage`, it just means you have two things called "validator", only one of which implements `Validator`.  Perhaps just leave them as is for now, something will come to us.

Reply:
If you don't mind I will rename this to `validation.InboundRejecter` / `OutboundRejecter` as originally requested.